tests/test_poll_alerts.py

```diff
@@ -0,0 +1,158 @@
+# coding: utf-8
+from cgi import test
+import os
+import yaml
+from fink_client.consumer import AlertConsumer
+from astropy.time import Time
+
+import skyportal_fink_client.utils.skyportal_api as skyportal_api
+
+# open yaml config file
+with open(
+    os.path.abspath(os.path.join(os.path.dirname(__file__))) + "/../config.yaml", "r"
+) as stream:
+    try:
+        conf = yaml.safe_load(stream)
+    except yaml.YAMLError as exc:
+        print(exc)
+
+
+def fid_to_filter(fid: int):
```

Comment:
Why this function is defined twice? (already in skyportal_fink_client/skyportal_fink_client.py)